src/tracing/librados.tp

```diff
@@ -2628,7 +2628,7 @@ TRACEPOINT_EVENT(librados, rados_watch3_enter,
     TP_FIELDS(
         ctf_integer_hex(rados_ioctx_t, ioctx, ioctx)
         ctf_string(oid, oid)
-        ctf_integer_hex(uint64_t, phandle, phandle)
+        ctf_integer_hex(uint64_t, phandle, phandle == NULL ? 0 : *phandle)
```

Comment:
I doubt anyone is using these tracepoints, but still ;)  `phandle` is an output parameter, so I think dumping the pointer in hex was intended (as opposed dumping to the pointee, which would probably be in decimal).  Similar to `ctf_integer_hex(rados_ioctx_t, ioctx, ioctx)` above, which is an opaque pointer (`typedef void *rados_config_t` in `src/include/rados/librados.h`) dumped in hex.